h5p-bildetema-words-grid-view/src/h5p/H5PWrapper.tsx

```diff
@@ -11,6 +11,7 @@ export type Params = {
   showWrittenWords?: boolean;
   showArticles?: boolean;
   l10n: Record<TranslationKey, string>;
+  onOpenDialog: (id: string) => void;
```

Comment:
Params relates to the attributes within semantics of each H5P. If you look at semantics.json for h5p-bildetema-words-grid-view you can see the fields for words, l10n, showWrittenWords and showArticles. So, onOpenDialog should be added further down in the components.

Reply:
Seems like you only need to use useChooseCollectionDialog in the necessary files, and not pass onOpenDialog trough App and H5PWrapper?

Reply:
If needed, a hook and context could be created like AudioContext.

Reply:
FIxed. Moved ChooseCollectionDialog to common folder and created context for Dialog. Updated to not pass it as a prop.